modules/transport.js

```diff
@@ -341,7 +344,12 @@ __private.hashsum = function (obj) {
 };
 
 __private.banPeer = function (options) {
+	if (!options.peer || !options.peer.ip || !options.peer.port) {
+		library.logger.trace ('Peer ban skipped', { options: options });
```

Comment:
Because we already have debug level here, for example:
```
[WRN] 2017-01-20 05:56:25 | ETIMEOUT Ban 213.222.208.29:8000 1 minutes POST http://213.222.208.29:8000/peer/blocks
[trc] 2017-01-20 05:56:25 | Peer banned - {"options":{"peer":{"ip":"213.222.208.29","port":8000,"string":"213.222.208.29:8000","state":2,"os":"linux3.16.0-4-amd64","version":"0.5.2","broadhash":"971a550df2470e4abb21b6a83474218d22b2d626c3197daeb6b4e3d1e7fc4c16","height":1864584},"code":"ETIMEOUT","req":{"url":"http://213.222.208.29:8000/peer/blocks","method":"POST","headers":{"os":"linux4.4.0-53-generic","version":"0.5.0","port":8666,"height":1864585,"nethash":"ed14889723f24ecc54871d058d98ce91ff2f973192075c0155ba2b7b70ad2511","broadhash":"51408e92aa77bbfef2d3b1b90a34d2869d86833a2111c6c79a00e75c2c51a10f","minVersion":"~0.5.0"},"timeout":5000,"body":{"block":{"version":0,"totalAmount":0,"totalFee":0,"reward":500000000,"payloadHash":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","timestamp":20782580,"numberOfTransactions":0,"payloadLength":0,"previousBlock":"4300873143868131230","generatorPublicKey":"3345cae6361e329bc931fda1245e263617c797c8b21b7abfb7914fcda1a7833b","transactions":[],"blockSignature":"39b62d50d4db90fc995e5a10b2a9220309381bbbb322053aea0ccbeae2c30e894e6e843b962b839c2be9aabcd85be0b8ac6d769f813c584ff2ca6bb7a2768d0c","id":"8494684807948542880","height":1864585,"secondsAgo":5.41,"fresh":true,"relays":2}}},"clock":60}}
[dbg] 2017-01-20 05:56:25 | Failed to broadcast to peer: 213.222.208.29:8000 - ETIMEOUT Request failed POST http://213.222.208.29:8000/peer/blocks
```